### DISCUSSION TITLE
Email integration

### DISCUSSION THREAD

**Discussion by mb720** [nvim-neorg/neorg#57] [Ideas]
Hi and thanks a lot for Neorg!

Are there plans to allow linking to (locally stored) emails?
Org mode for Emacs [supports this](https://orgmode.org/manual/External-Links.html) apparently (I've never tried it).

It sure would be quite useful for me to link from a file with notes to my emails. I currently use Mutt and OfflineIMAP for mails.

**Comment by vhyrro** [comment]
To be honest i have never tried it either hah

I don't know when this feature would come exactly tbh (we're busy dealing with making features for 0.1 and preparing workflow ideas for 0.2). I wish there were some sort of really good lua email plugin for neovim that we could easily interface with alongside a fallback for other mail clients. Now *that* would be awesome :)
